src/title.rs

```diff
@@ -1,25 +1,76 @@
+#![warn(missing_docs)]
+//! This module holds the [Title] element and its related configuration types.
+//! A title is a piece of [Block](crate::widgets::Block) configuration.
+
 use strum::{Display, EnumString};
 
 use crate::{layout::Alignment, text::Line};
 
+/// A block title
+///
+/// It can be aligned (see [Alignment]) and positioned (see [Position]).
+///
+/// # Example
+///
+/// ```
```

Comment:
For the example it's worth noting that a title can have multiple Spans (because it takes a line), which will can each be formatted differently. This is particularly useful for highlighting shortcut keys. E.g. `Line::from(vec!["Q".white(), "uit".gray()]` might be a good example.